pkg/cmd/flagtypes/addr.go

```diff
@@ -106,6 +106,8 @@ func (a *Addr) Set(value string) error {
 
 	if !a.AllowPrefix {
 		addr.Path = ""
+	} else if addr.Path == "/" {
```

Comment:
It's important to normalize the original value we get, so it doesn't result in duplicates in config depending on how it was provided to the command, for example.